pkg/pod/container.go

```diff
@@ -113,6 +113,81 @@ func (builder *ContainerBuilder) WithSecurityCapabilities(sCapabilities []string
 	return builder
 }
 
+// WithDropSecurityCapabilities drops SecurityCapabilities from the container definition.
+func (builder *ContainerBuilder) WithDropSecurityCapabilities(sCapabilities []string, redefine bool) *ContainerBuilder {
+	glog.V(100).Infof("Dropping a list of SecurityCapabilities %v from container %s",
+		sCapabilities, builder.definition.Name)
+
+	if !areCapabilitiesValid(sCapabilities) {
+		glog.V(100).Infof("Given SecurityCapabilities %v are not valid. Valid list %s",
+			sCapabilities, AllowedSCList)
+
+		builder.errorMsg = "one of the give securityCapabilities is invalid. Please extend allowed list or fix parameter"
```

Comment:
's/give/provided/'

"the allowed list"

Reply:
thanks, updated